e2e/lib/testing.sh

```diff
@@ -12,6 +12,8 @@
 # shellcheck source=e2e/lib/_utils.sh
 source "${E2EDIR:-$HOME/test-infra/e2e}/lib/_utils.sh"
 
+LOG_DIR="/var/log/e2e"
```

Comment:
Looks like a permissions issue on this path

Reply:
@efiacor you're right, good catch, thanks